pkgs/applications/window-managers/sway/default.nix

```diff
@@ -13,21 +13,15 @@
 , trayEnabled ? systemdSupport && dbusSupport
 }:
 
-# The "sd-bus-provider" meson option does not include a "none" option,
-# but it is silently ignored iff "-Dtray=disabled".  We use "basu"
-# (which is not in nixpkgs) instead of "none" to alert us if this
-# changes: https://github.com/swaywm/sway/issues/6843#issuecomment-1047288761
-assert trayEnabled -> systemdSupport && dbusSupport;
-let sd-bus-provider = if systemdSupport then "libsystemd" else "basu"; in
-
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "sway-unwrapped";
   version = "1.8.1";
 
+  inherit enableXWayland systemdSupport isNixOS;
```

Comment:
Optional: We could also inherit `trayEnabled` and `dbusSupport` for consistency.